draft-ietf-quic-transport.md

```diff
@@ -1433,8 +1432,13 @@ earlier Version Negotiation packet. A client MUST discard a Version Negotiation
 packet that lists the QUIC version selected by the client.
 
 How to perform version negotiation is left as future work defined by future
-versions of QUIC.  In particular, that future work will ensure robustness
-against version downgrade attacks; see {{version-downgrade}}.
+standards-track specifications.  In particular, that future work will
+ensure robustness against version downgrade attacks; see
+{{version-downgrade}}.  New versions of QUIC (including
+non-standards-track versions) might be specified prior to the
+availability of a version negotiation mechanism.  Until a version
+negotiation mechanism is available, new versions of QUIC MUST respond to
+Version Negotiation packets as specified above.
```

Comment:
I don't think that we need to include these last two sentences.  While we've definitely strayed into making forward-looking statements (read: predictions) about what we might do, this goes too far in my opinion.

Reply:
I concede it was a bit speculative to say that this reflects existing consensus.  That said, there is perhaps a spectrum of options, from removing it entirely to "are expected to", "SHOULD", etc.  It sounds like you prefer just removing it entirely, to confirm?

Reply:
Yes.  I don't think that we get much value from saying anything like this, regardless of strength, simply because it is making a prediction about work that might happen in the future.

Reply:
Fair enough.  I will wait a bit before pushing an update to effectuate that, in case others want to comment.